crypto/src/key/rschnorr/mod.rs

```diff
@@ -290,4 +290,19 @@ mod test {
         let sig = sk.sign_message(&mut rng, &msg).unwrap();
         assert!(pk.verify_message(&sig, &msg));
     }
+
+    #[test]
+    fn sk_zeroed() {
+        use std::slice;
+        let mut rng = rand::rngs::StdRng::from_entropy();
+        let zero_sk = &vec![0u8; 32][..];
+        unsafe {
+            let hldr;
+            {
+                let (sk, _pk) = MLRistrettoPrivateKey::new(&mut rng);
+                hldr = sk.as_bytes().as_ptr();
+            }
+            assert_eq!(slice::from_raw_parts(hldr, 32), zero_sk);
+        }
```

Comment:
Which parts are actually unsafe? It's only `from_raw_parts` as far as I can see. Usually, the practice is for the scope of `unsafe` to be minimized.

Reply:
Yep it's just `from_raw_parts`